eth/main.cpp

```diff
@@ -406,403 +320,601 @@ int main(int argc, char** argv)
+            ("config", po::value<string>(), "<file> Configure specialised blockchain using given JSON information.")
+            ("oppose-dao-fork", "Ignore DAO hard fork (default is to participate).\n")
+            ("mode,o", po::value<string>(), "<full/peer>  Start a full node or a peer node (default: full).\n")
+            ("json-rpc,j", "Enable JSON-RPC server (default: off).")
+            ("ipc", "Enable IPC server (default: on).")
+            ("ipcpath", po::value<string>(), "Set .ipc socket path (default: data directory)")
+            ("admin-via-http", "Expose admin interface via http - UNSAFE! (default: off).")
+            ("no-ipc", "Disable IPC server.")
+		    ("json-rpc-port", po::value<int>(), "See description under.")
+		    ("rpccorsdomain", po::value<string>(), "<domain>  Domain on which to send Access-Control-Allow-Origin header.")
+		    ("admin", po::value<string>(), "<password>  Specify admin session key for JSON-RPC (default: auto-generated and printed at start-up).")
+		    ("kill,K", "Kill the blockchain first.")
+		    ("kill-blockchain", "Kill the blockchain first.")
+		    ("rebuild,R", "Rebuild the blockchain from the existing database.")
+		    ("rescue", "Attempt to rescue a corrupt database.\n")
+		    ("import-presale", po::value<string>(), "<file>  Import a pre-sale key; you'll need to specify the password to this key.")
+		    ("import-secret,s", po::value<string>(), "<secret>  Import a secret key into the key store.")
+		    ("import-session-secret,S", po::value<string>(), "<secret>  Import a secret session into the key store.")
+		    ("master", po::value<string>(), "<password>  Give the master password for the key store. Use --master \"\" to show a prompt.")
+		    ("password", po::value<string>(), "<password>  Give a password for a private key.\n")
+		    ("extra-data", po::value<string>(), "Set extra data")
+		    ("genesis", po::value<string>(), "Set genesisJSON")
+		    ("genesis-json", po::value<string>(), "Set genesisJSON")
+		    ("json-admin", po::value<string>(), "Set jsonAdmin")
+    ;
+	po::options_description clientTransacting("Client transactions");
+	clientTransacting.add_options()
+			("ask", po::value<string>(), "See description under.")
+			("bid", po::value<string>(), "See description under.")
+			("unsafe-transactions", "Allow all transactions to proceed without verification. EXTREMELY UNSAFE.")
+	;
+	po::options_description clientMining("Client mining");
+	clientMining.add_options()
+			("address,a", po::value<string>(), "<addr>  Set the author (mining payout) address to given address (default: auto).")
+			("author", po::value<string>(), "<addr>  Set the author (mining payout) address to given address (default: auto).")
+			("mining,m", po::value<string>(), "<on/off/number>  Enable mining, optionally for a specified number of blocks (default: off).")
+			("force-mining,f", "Mine even when there are no transactions to mine (default: off).")
```

Comment:
This can be deleted

Reply:
Why?

Reply:
all this 4 options?

Reply:
Only `-f` & `--force-mining`
mining itself is still supported